docs/post-response-codes.md

```diff
@@ -0,0 +1,80 @@
+# PostResponseCodes
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI (swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Async-V1-11
+
+## Output Message
+
+Synchronous POST operations must have responses with 200, 204 and default return codes. They also must not have other response codes.
+Long-running POST operations must have responses with 200, 202 and default return codes. They also must not have other response codes.
```

Comment:
>responses with 200, 202 [](http://example.com/codeflow?start=39&length=23)

must have 202 and default response codes. They also should have a 200 response code to indicate the schema for the final response schema if the final response is intended to have a schema. If the final response schema is empty the 200 response code must not be specified.